### PR TITLE
Refactor list commands to use share common parent class

### PR DESCRIPTION
**Pull request by BartoszCki**

### CODE REVIEW

File: paperspace/commands/deployments.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
-import pydoc
-
 import terminaltables
 
-from paperspace import config, version, client, logger
-from paperspace.commands import CommandBase
-from paperspace.utils import get_terminal_lines
+from paperspace import config, version, client
+from paperspace.commands import common
+from paperspace.commands.common import CommandBase
```
Comment (kossak) [inline paperspace/commands/deployments.py]: why do you keep importing both `common` and `from common`?

File: paperspace/commands/jobs.py [hunk 2]
```diff
@@ -4,9 +4,10 @@
 from click import style
 
 from paperspace import config, client
-from paperspace.commands import CommandBase
+from paperspace.commands.common import CommandBase
 from paperspace.utils import get_terminal_lines
 from paperspace.workspace import S3WorkspaceHandler
+from . import common
```
Comment (kossak) [inline paperspace/commands/jobs.py]: same as above. also, why one time it's relative, other time it's absolute?

File: paperspace/commands/deployments.py (hunk 1 above)
Comment (BartoszCki) [reply paperspace/commands/deployments.py]: Missed that, fixed now

File: paperspace/commands/jobs.py (hunk 2 above)
Comment (BartoszCki) [reply paperspace/commands/jobs.py]: fixed

**Pull request merged**